webApps/apps.json

```
@@ -198,6 +198,53 @@
           "url": "https://github.com/owncloud/web-extensions/tree/main/packages/web-app-unzip"
         }
       ]
+    },
+    {
+      "id": "com.github.jankaritech.mdpresentation-viewer",
+      "name": "Markdown Presentation Viewer",
+      "subtitle": "Convert and view markdown file (.md) as presentation.",
+      "license": "AGPL-2.0",
+      "versions": [
+        {
+          "version": "2.0.0",
+          "url": "https://github.com/JankariTech/web-app-presentation-viewer/releases/download/2.0.0/mdpresentation-viewer-2.0.0.zip"
+        },
+        {
+          "version": "1.0.0",
+          "url": "https://github.com/JankariTech/web-app-presentation-viewer/releases/download/1.0.0/mdpresentation-viewer-1.0.0.zip"
+        }
+      ],
+      "authors": [
+        {
+          "name": "Jankari Tech",
```

```suggestion
          "name": "JankariTech",
```

We spell the company name without space in the international context. Only in Nepal we have to add the space 🙈